backend/inspirehep/migrator/tasks.py

```diff
@@ -109,6 +110,37 @@ def migrate_record_from_legacy(recid):
     db.session.commit()
 
 
+def migrate_from_mirror_run_step(
+    disable_orcid_push=True, disable_references_processing=True, step_no=1
+):
+    """Allows to easily run step by step migration only for valid records """
+    if step_no == 0:
+        query = LegacyRecordsMirror.query.with_entities(LegacyRecordsMirror.recid)
+        query = query.filter(LegacyRecordsMirror.valid.is_(True))
+    elif 0 < step_no < 3:
+        query = PersistentIdentifier.query.with_entities(
+            PersistentIdentifier.object_uuid
+        )
+    else:
+        print("Wrong step number!")
```

Comment:
can we use ``echo``? also more info will be better like, the step etc

Reply:
Actually for all tasks in migrator we are using print.

Reply:
amazing :D Let's change that bit by bit

Reply:
And I don't see why i should print step number to this message. It will be the same number provided by user few seconds before displaying this message)

Reply:
I changed all prints to echo in this file